### ISSUE TITLE
Great oportunity for linked brushing and gmap integration

### ISSUE THREAD

**Issue by paddymul** [bokeh/bokeh#875]
I was reading this article http://www.citylab.com/housing/2014/07/cost-of-living-is-really-all-about-housing/373128/ . and the plots were really hard to follow.

If you combined the gmap plot of metro areas with a scatter plot that had an x-axis of cost of goods , and a y-axis of cost of rent, you could build some really neat linked brushing examples.  

Does linked brushing work for gmap overlays?

**Comment by birdsarah** [comment]
@paddymul - this works nicely - see http://nbviewer.ipython.org/github/birdsarah/bokeh-miscellany/blob/master/tests/875_linked_brushing_with_gmap.ipynb

@bryevdv - do we want a version of this - or maybe something a bit more real-world (like paddy's suggestion) in the examples?

**Comment by birdsarah** [comment]
Discussion is now more than 3 months old. Can re-open if renewed interest.
